logger.hh

```diff
@@ -1,18 +1,8 @@
+#pragma once  // REFACTOR: Replaced include guards with modern pragma once
```

Comment:
please move down to where the `#ifndef` include guard was and remove REFACTOR comment